modules/localStorage.js

```diff
@@ -0,0 +1,100 @@
+const entryTask = document.getElementById('newTask');
+
+const storageAvailable = (type) => {
+  try {
+    const storage = window[type];
+    const x = '__storage_test__';
+    storage.setItem(x, x);
+    storage.removeItem(x);
+    return true;
+  } catch (e) {
+    return e instanceof DOMException && (
+    // everything except Firefox
+      e.code === 22
+    // Firefox
+    || e.code === 1014
+    // test name field too, because code might not be present
+    // everything except Firefox
+    || e.name === 'QuotaExceededError'
+    // Firefox
+    || e.name === 'NS_ERROR_DOM_QUOTA_REACHED');
+  }
+};
+
+export const savedata = () => {
+  const listOnStorage = JSON.parse(localStorage.getItem('toDoList')) || [];
```

Comment:
-An export is needed when your code is separated into separate modules, This as you can see has real function here kindly erase and write function directly

Reply:
'localStorage' is in separate modules. I import it into my 'index.js'. so it's necessary to write 'export' here. What do you think about this? Should I have to merge this separate module in my 'index.js'?